src/Sentry/SentryMonitorOptions.cs

```diff
@@ -26,6 +26,11 @@ public class SentryMonitorOptions : ISentryJsonSerializable
     /// <param name="cronTab"></param>
     public void Interval(string cronTab)
     {
+        if (_type is not SentryMonitorScheduleType.None)
+        {
+            throw new ArgumentException("You tried to set the interval twice. The Check-Ins interval is supposed to be set only once.");
+        }
+
         _type = SentryMonitorScheduleType.Crontab;
         _crontab = cronTab;
```

Comment:
According to [our docs](https://develop.sentry.dev/sdk/check-ins/#using-crontab):
> The interval unit. Should be one of year, month, week, day, hour, minute.

I think we should design our API so that people can't misconfigure this then. One way to validate the `cronTab` argument would be to use regular expressions. Alternatively, it doesn't necessarily have to be a string... we have to send a string to Sentry (as part of the protocol) but that might not be the best way for users to configure this via our SDK.

Reply:
Fair point. I added some validation to the crontab.